### ISSUE TITLE
improve dropdown menu

### ISSUE THREAD

**Issue by mattl** [creativecommons/mp#17]
use <optgroup> to have full copyright (at the bottom) then work back up from BY-NC-ND to 0 with groups for public domain and free culture licenses.

**Comment by robmyers** [comment]
This is a bit of a pain to implement (the current logic is simple, and goes with the Those Numbers order for the licenses). Can I just re-order them?

Also we don't currently support PDM, "No known copyright restrictions", or US Govt. work. Should we? Again, I'd rather not.

However, if we do add the divisions:

```
----Public Domain
CC0
---Free Culture Licenses
BY
BY-SA
---Proprietary Licenses
BY-NC
BY-NC-SA
BY-ND
BY-NC-ND
----No License
All Rights Reserved
```

**Comment by robmyers** [comment]
Implemented in af84b97 . Let me know if any of the labels or ordering need changing.
